src/components/includes/includes--footer-social.njk

```diff
@@ -0,0 +1,65 @@
+<div aria-labelledby="social-media-heading" class="rvt-c-footer-social" role="complementary">
```

Comment:
This doesn't need to hold things up right on this PR for now either, but the `rvt-c-...` convention was only used to denote that this was a "custom" component/implementation and not a built in part of the rivet 2 alpha we were using.

All the `-c-` suffixes can be removed when we move into the real code base. 👍🏻  I think we can tackle this as a separate refactoring pass/PR though.